### DISCUSSION TITLE
48

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#708] [Comments]
http://jojoldu.tistory.com/48

**Comment by jojoldu** [comment]
작성자: visualkhh

찾던거입니다~~~~~~~~~~^^ 공유 감사합니다
즐거운하루되세요

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

도움이 되셨다니 다행입니다!
방문해주셔서 감사합니다^^

**Comment by jojoldu** [comment]
작성자: velocity

velocity 의 경우 application.properties 파일의 설정으로 cache=false 를 할수 없는데 그러면 해당 기능은 사용 못하는건가요?...

**Comment by jojoldu** [comment]
작성자: 만득이

와우~ 마지막 두가지 설정을 하니까 잘되네요~
감사합니다용~

**Comment by jojoldu** [comment]
작성자: 곰길이

삽질 하다가 이글 보고 살았네요. 감사합니다!

**Comment by jojoldu** [comment]
작성자: alicek106

감사합니다. 링크로 퍼갈게요 ^^;; 분명히 나중에 또 볼것 같네요.

**Comment by brainbackdoor** [comment]
잘 읽었습니다. 감사합니다 ㅎ

**Comment by ocharming** [comment]
intellij 정적 리소스 reload가 안되서 너무 불편합니다... 몇일째 적용이 안되고 있는데 저만 안되는 건가요ㅠㅠ?

**Comment by csbok** [comment]
https://lhb0517.tistory.com/entry/SpringIntelliJ-SpringBoot-HotSwap-with-IntelliJ
버전이 올라가면서 바뀐것 같습니다. 이 글의 추가 내용 보고 적용했습니다.

**Comment by jojoldu** [comment]
@csbok 좋은글 감사합니다 :)

**Comment by MARI2020201101** [comment]
계속 인텔리제이 설정으로 끙끙앓고 있었는데!! 너무 감사합니다. 정말 많은 도움이 되었어요.

**Comment by hello-yoochul** [comment]
참고 잘했습니다~

compiler.automake.allow.when.app.running 가 인텔리제이에서 없어지고 설정에 advanced setting 으로 갔다고 하네요. 

https://youtrack.jetbrains.com/issue/IDEA-274903/In-IntelliJ-20212-compilerautomakeallowwhenapprunning-disappear-Unable-to-enable-live-reload-under-Spring-boot
